timesketch/frontend-ng/src/components/LeftPanel/SearchTemplates.vue

```diff
@@ -80,6 +80,9 @@ export default {
     ApiClient.getSearchTemplates()
       .then((response) => {
         this.searchtemplates = response.data.objects[0]
+        if (typeof(this.searchtemplates) === 'undefined') {
+          this.searchtemplates = []
+        }
```

Comment:
Should we return an empty list from the backend API instead?

Reply:
![image](https://user-images.githubusercontent.com/741037/201303306-494ebe93-62d6-4b1d-af9a-3978d8b14c9e.png)

Reply:
So I just looked at https://github.com/jaegeral/timesketch/blob/2022-11-09_2395/timesketch/api/v1/resources/scenarios.py#L73 and we do return an undefined in many more places, so I suggest we accept this change and make a bigger effort to make all the API endpoints behave similar. WDYT?

Reply:
I'm working on that code elsewhere, and it is being revamped. So, yeah SG, I'll fix it later.